docs/UserGuide.md

```diff
@@ -99,15 +97,21 @@
 
    ![alt text](./images/userguide/event_list.png "")
 
-10. Let's view the index of task. Enter `i`.
+10. All task in the database will be assigned a unique index. The unique index can be seen to the left of the task name.
 
-    ![alttext](./images/userguide/index.png "")
-
-    Enter anything into command line to hide index view.
+    ![alttext](./images/userguide/index.png "")to update with actual app shots
 
 11. To delete a task, try:
+    	
+        del float 1
```

Comment:
This will unfortunately require some code restructuring in the parser :-(. (Waiting on @nightism for the parser rework :-) )

Can we go with `del-float` and `edit-float` for now?